quinn-proto/src/tests/token.rs

```diff
@@ -23,6 +23,40 @@ fn stateless_retry() {
     assert_eq!(pair.server.known_cids(), 0);
 }
 
+#[test]
+fn stateless_retry_token_expired() {
```

Comment:
Could you rename to avoid usage of the phrase "stateless retry", and just call it a retry token? [I've been pushing Quinn to avoid that](https://github.com/quinn-rs/quinn/pull/2086), because I think it's too easy to confuse with stateless resets, and it's not a phrase that's used in RFC 9000.

Reply:
Thank you for your information! I will rename this testcase.